src/data/systeminfo.rs

```diff
@@ -121,7 +121,17 @@ impl CollectData for SystemInfo {
 
         match rt.block_on(EC2Metadata::get_instance_metadata()) {
             Ok(s) => self.set_instance_metadata(s),
-            Err(e) => error!("An error occurred: {}", e),
+            Err(e) => {
+                warn!("An error occurred: {}", e);
```

Comment:
What is it, a warning or an error? :p

"Unable to get instance metadata: {}" would be more meaningful since there isn't going to be much context to understand what was being attempted when the failure is reported.